Packs/SuspiciousDomainHunting/Playbooks/playbook-Suspicious_Domain_Hunting_Incident_Handling.yml

```diff
@@ -0,0 +1,1395 @@
+      version: -1
+      name: Check if a malicious Domain was identified
+      description: ""
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "30"
+      Malicious:
+      - "33"
+      Suspicious:
+      - "39"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Domain(val.Malicious).Name
+            iscontext: true
+          right:
+            value: {}
+        - operator: greaterThanOrEqual
+          left:
+            value:
+              simple: HTMLSimilarity.SimilarityPercentage
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: inputs.similarityMaxThreshold
+            iscontext: true
+        - operator: greaterThanOrEqual
+          left:
+            value:
+              simple: ImageSimilarity.SSIM
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: inputs.similarityMaxThreshold
+            iscontext: true
+        - operator: lessThanOrEqual
+          left:
+            value:
+              simple: ImageSimilarity.MSE
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: inputs.similarityMaxThreshold
+            iscontext: true
+    - label: Suspicious
+      condition:
+      - - operator: InRange
+          left:
+            value:
+              complex:
+                root: HTMLSimilarity
+                accessor: SimilarityPercentage
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: inputs.similarityMinThreshold
+                transformers:
+                - operator: concat
+                  args:
+                    prefix: {}
+                    suffix:
+                      value:
+                        simple: ','
+                - operator: concat
+                  args:
+                    prefix:
+                      iscontext: true
+                    suffix:
+                      value:
+                        simple: ${inputs.similarityMaxThreshold}
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 817.5,
+          "y": 595
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "14_24_#error#": 0.12,
+      "24_30_Benign": 0.49,
+      "29_22_yes": 0.44,
+      "2_24_#error#": 0.24,
+      "35_21_yes": 0.55
+    },
+    "paper": {
+      "dimensions": {
+        "height": 2730,
+        "width": 2285,
+        "x": -427.5,
+        "y": -890
+      }
+    }
+  }
+inputs:
```

Comment:
In the layout, you used the input 'contactEmail' in the 'ask for takedown' button. Please include this input in the playbook.